Dark_Heresy_2ed/DarkHeresy2ed.html

```diff
@@ -2516,12 +2516,12 @@ <h3>Ranged Weapons</h3>
 							<div class="sheet-item" style="width: 15%;"><label>Reload:</label></div>
 							<div class="sheet-item" style="width: 15%;"><input name="attr_Rangedweaponreload" type="text" /></div>
-							<div class="sheet-item" style="width: 16%;"><button name="roll_Rangedhit" type="roll" value="&{template:dh2ed}[[1 + floor(abs([[@{BallisticSkill}+[[?{Aim | No aim (+0),+0[no aim] | Half aim (+10),+10[half aim] | Full aim (+20),20+[full aim]}?{Range | Point Blank (+30),+30[point blank] | Short Range (+10),+10[short] | Standard range (+0),+0[standard range] | Long Range (-10),-10[long range] | Extreme Range (-30),-30[extreme range]}?{Rate of Fire/Attack Type| Standard (+10),+10[standard] | Semi auto (+0),+0[semi-auto] | Full Auto (-10),-10[full-auto] | Called Shot (-20),-20[called-shot] | Suppressing Fire (-20),-20[suppressing]}+[[?{Modifier|0}]]]]]]-[[[[1d100cs<1cf>100]]]])/10)]]{{character=@{character_name}}}{{name=@{Rangedweaponname}}}{{roll= $[[3]] }}{{degrees=$[[5]]}}{{skill=@{BallisticSkill}}}{{modifier=$[[1]] }}{{target=$[[2]]}}"> <label>Hit</label> </button></div>
+							<div class="sheet-item" style="width: 16%;"><button name="roll_Rangedhit" type="roll" value="&{template:dh2ed}[[1 + floor(abs([[@{BallisticSkill}+[[?{Aim | No aim (+0),+0[no aim] | Half aim (+10),+10[half aim] | Full aim (+20),+20[full aim]}?{Range | Point Blank (+30),+30[point blank] | Short Range (+10),+10[short] | Standard range (+0),+0[standard range] | Long Range (-10),-10[long range] | Extreme Range (-30),-30[extreme range]}?{Rate of Fire/Attack Type| Standard (+10),+10[standard] | Semi auto (+0),+0[semi-auto] | Full Auto (-10),-10[full-auto] | Called Shot (-20),-20[called-shot] | Suppressing Fire (-20),-20[suppressing]}+[[?{Modifier|0}]]]]]]-[[[[1d100cs<1cf>100]]]])/10)]]{{character=@{character_name}}}{{name=@{Rangedweaponname}}}{{roll= $[[3]] }}{{degrees=$[[5]]}}{{skill=@{BallisticSkill}}}{{modifier=$[[1]] }}{{target=$[[2]]}}"> <label>Hit</label> </button></div>
 						</div>
 						<div class="sheet-row">
 							<div class="sheet-item" style="width: 15%;"><label>Special:</label></div>
 							<div class="sheet-item" style="width: 69%;"><input name="attr_Rangedweaponspecial" type="text" /></div>
-							<div class="sheet-item" style="width: 16%;"><button name="roll_Rangeddamage" type="roll" value="&{template:dh2ed-dmg}{{character=@{character_name}}}{{name=@{Rangedweaponname}}}{{damage=[[@{Rangedweapondamage}+@{UnS}+floor(@{strength}/10)]]}}{{damagetype=@{Rangedweapontype}}}{{special=@{Rangedweaponspecial}}}{{penetration= @{Rangedweaponpen}}}"> <label>Damage</label> </button></div>
+							<div class="sheet-item" style="width: 16%;"><button name="roll_Rangeddamage" type="roll" value="&{template:dh2ed-dmg}{{character=@{character_name}}}{{name=@{Rangedweaponname}}}{{damage=[[@{Rangedweapondamage}]]}}{{damagetype=@{Rangedweapontype}}}{{special=@{Rangedweaponspecial}}}{{penetration= @{Rangedweaponpen}}}"> <label>Damage</label> </button></div>
```

Comment:
I still seem to have the inclusion of Strength to ranged weapon damage in my campaign. do you know how to remedy this?

https://app.roll20.net/forum/post/9601894/dark-heresy-2nd-edition-ranged-weapons

Issue still seems to be present for others also

Reply:
Sorry for the delayed reply,
it seems like the fix was unintentionally removed in a later pull request.
I have proposed another fix.